### ISSUE TITLE
fix phpmyadmin install

### ISSUE THREAD

**Issue by vivithemage** [vivithemage/do-wp#15]
PHPadmin install has been started but it is not completed.

**Comment by robinzigmond** [comment]
I'm going to close this as the tool now installs phpmyadmin - but note that when you try to access a database table on the phpmyadmin panel, it shows an error and spews out quite a bit of detail (which I can't really make sense of). However you can click to ignore this and everything then works fine. Still might be worth some investigation and possible fixing at a later date.

**Comment by vivithemage** [comment]
Thanks Robin,
I'd suggest leaving this open for now. It would be good to know what those errors are and how we can solve them.

**Comment by robinzigmond** [comment]
I looked at this again and managed to fix it. Fairly simple to do in the end, I just used what was given in this stack overflow thread where there is a long discussion of it. It's pretty unbelievable that there can be such a stupid error in a release of phpmyadmin, especially one that's so simple to fix and apparently still hasn't been! https://stackoverflow.com/questions/48001569/phpmyadmin-count-parameter-must-be-an-array-or-an-object-that-implements-co
